src/components/Loading/README.md

```diff
@@ -32,7 +32,13 @@ Use to indicate that the page is loading.
 
 ## Required components
 
-The loading component must be wrapped in the [frame](/components/structure/frame) component.
+The loading component must be wrapped in the [frame](/components/structure/frame) component unless used in an embedded application.
+
+---
+
+## Use in an embedded application
+
+Passing an API key to the [app provider component](https://polaris.shopify.com/components/structure/app-provider#section-initializing-the-shopify-app-bridge) causes the loading component to delegate to the [Shopify App Bridge](https://help.shopify.com/en/api/embedded-apps/app-bridge) instead of rendering as it would in a stand-alone application.
```

Comment:
Can you elaborate on what this actually means? I think a code example of how it should be used in an embedded application would be helpful.

Reply:
For instance, the App Bridge docs instruct us to use the package's `Loading` action, and following the example in the `AppProvider` docs, it follows that you need to do the following: 

```jsx
import {Loading} from '@shopify/app-bridge/actions';
...

startLoading = () => {
  const loading = Loading.create(this.context.polaris.appBridge);
  loading.dispatch(Loading.Action.START);
}
```

But this will cause their Polaris app to not compile.